js/index.js

```diff
@@ -13,4 +13,26 @@ for (const item of skills){
     skill.innerText = item;
     skillsList.appendChild(skill);
 }
-    
+let messageSection = document.getElementById('messages');
+let messageList = messageSection.querySelector('ul');
+let newMessage = document.createElement('li');
+newMessage.getElementByName('usersName').innerHTML = "<a href=mailto: + newMessage.getElementByName('usersEmail').email>";
```

Comment:
This needs to be inside the submit even handler.  The new message content is not defined here.

Reply:
newMessage should be a new li element created every time the submit button is used. (document.createElement('li'))